src/Sylius/Bundle/UiBundle/Resources/translations/messages.it.yml

```diff
@@ -872,7 +872,7 @@ sylius:
         view_more: 'Dettaglio'
         view_product_details: 'Mostra dettagli %product%'
         view_your_order_or_change_payment_method: 'Mostra l''ordine o cambia il metodo di pagamento'
-        view_your_store: 'Toon jouw winkel'
+        view_your_store: 'Vai al tuo store'
```

Comment:
We are using Crowdin for updating translations, instead of changing them in PRs. Please revert this one :)
http://docs.sylius.org/en/latest/contributing/translations/index.html

Reply:
Hi @CoderMaggie, 
for the translation ok, I reverted the commit.
For the getNewLocaleCode() method, I'm not sure that allow setting a different default locale that "en_ES" is a good idea... I'm testing this and it seems that we need to have the local default as en_US :(